doc-src/templates/default/fulldoc/html/js/full_list.js

```diff
@@ -0,0 +1,246 @@
+(function() {
```

Comment:
This file comes from https://github.com/lsegal/yard/blob/main/templates/default/fulldoc/html/js/full_list.js but has some modifications.

Here is the diff:

```
64a65,68
>     // custom - flip aria attribute
>     $(this).attr('aria-expanded', function (i, attr) {
>         return attr == 'true' ? 'false' : 'true'
>     });
66a71,84
> 
>   // custom - needed for accessibility
>   $('#full_list a.toggle').on('keypress',function(evt) {
>     // enter key
>     if (evt.which == 13) {
>       evt.stopPropagation();
>       evt.preventDefault();
>       $(this).parent().parent().toggleClass('collapsed');
>       $(this).attr('aria-expanded', function (i, attr) {
>           return attr == 'true' ? 'false' : 'true'
>       });
>       highlight();
>     }
>   });
94c112,113
<   $('#full_list').after("<div id='noresults' style='display:none'></div>");
---
>   // custom - add role status
>   $('#full_list').after("<div id='noresults' role='status' style='display:none'></div>");
144a164
> 
157c177,179
<   if ($('#full_list li:visible').size() === 0) {
---
>   // bug fix
>   var found = $('#full_list li > div.found').length;
>   if (found === 0) {
160c182,183
<     $('#noresults').text('').hide();
---
>     // accessibility - read out results
>     $('#noresults').text('There are ' + found + ' results.').hide().fadeIn();
190a214,220
> 
>   // custom - flip aria attribute
>   $target.find('a.toggle').attr('aria-expanded', 'true')
>   $target.parentsUntil('#full_list', 'li').each(function(i, el) {
>     $(el).find('> div > a.toggle').attr('aria-expanded', 'true');
>   });
> 
216c246
< })();
---
> })();
\ No newline at end of file
```

Reply:
Maybe add a comment at the top of the file about the source.

Reply:
Done